code/modules/mob/living/living.dm

```diff
@@ -17,6 +17,8 @@
 	medhud.add_to_hud(src)
 	faction += "[REF(src)]"
 	GLOB.mob_living_list += src
+	if(movement_type & FLYING)
```

Comment:
I feel it would be easier to drop support for the movement_type variable here than keep all these. Is anything that's not living using this anyway ?

Reply:
Yeah, i suppose it's no longer needed. I still need something to initialize the flying trait though.